### ISSUE TITLE
New "imaging-spectroscopy" branch: required steps before merging with the main branch

### ISSUE THREAD

**Issue by afbattaglia** [i4Ds/STIX-GSW#173]
As discussed with @samaloney (thanks for creating the new branch!), my plan is to work on the STIX imaging-spectroscopy software on a separate branch. We need to temporarily modify some procedures to ensure that the software works as intended. The issue is explained in detail in #154. Once this issue is fixed and merged into the main branch, I can modify the imaging-spectroscopy software to ensure compatibility with the new solution.

The purpose of this issue is to list all the necessary tasks that have to be completed before merging the imaging-spectroscopy branch with the main branch:
- [ ] solve issue #154. We already discussed this with @grazwegian and we are working on this;
- [ ] test the STIX imaging spectroscopy software on different cases:
  - [ ] single compact source;
  - [ ] double compact sources;
  - [ ] one compact source and one extended source;
- [ ] test the procedures for the electron maps and the regularized visibilities. The STIX imaging-spectroscopy software is intended to work with the regularized visibilities as well. We are collaborating with @annavolp to accomplish this;
- [ ] once we are satisfied with all the tests, we should place all necessary procedures in the appropriate folders. Initially, I would create a separate folder for easier and faster access.

Once all of the above tasks are completed, we will be ready to merge the imaging-spectroscopy branch into the main branch.

**Comment by ennosigaeus** [comment]
We discussed this today (Paolo, Muriel, and I) and we see that this seems to be linked to the branch imaging-spectroscopy and the PR #174 (which is a branch forked from imaging-spectroscopy). We don't quite know what direction to take from here. We need to discuss this with the larger Monday group.
